GANDLF/parseConfig.py

```diff
@@ -1,35 +1,42 @@
 import sys, yaml, pkg_resources
 
-## dictionary to define defaults for appropriate options
+## dictionary to define defaults for appropriate options, which are evaluated
 parameter_defaults = {
     "weighted_loss": False, # whether weighted loss is to be used or not
     "verbose": False, # general application verbosity
     "q_verbose": False, # queue construction verbosity
     "medcam_enabled": False, # interpretability via medcam
     "save_output": False, # save outputs during validation/testing
-    "in_memory": False, # pin data to gpu memory
-    "patch_sampler": "uniform", # type of sampling strategy
+    "in_memory": False, # pin data to cpu memory
+    "in_memory_gpu": False, # pin data to gpu memory 
```

Comment:
I am pretty sure we are not using DALI or any in_memory_gpu supporting libraries.

Reply:
Actually, this is an option for this: https://pytorch.org/docs/stable/data.html#memory-pinning